README.md

```diff
@@ -1,15 +1,15 @@
-<p align="center">
-    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" />
+<p style="text-align: center; ">
+    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" alt="Laravel Zero Logo" />
 </p>
 
-<p align="center">
-  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></img></a>
+<p style="text-align: center; ">
+  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></a>
```

Comment:
Although it does work to not have a closing tag, it's probably good to keep a self-closing tag.
```suggestion
  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"/></a>
```

Reply:
I'm cool with that if you guys are. I prefer the self closing tag, but I was trying to keep it consistent with the following `<img>` tags.